policy-service/src/unit-tests/java/uk/gov/gchq/palisade/service/policy/request/SetTypePolicyRequestTest.java

```diff
@@ -18,41 +18,33 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
 
 import uk.gov.gchq.palisade.User;
 import uk.gov.gchq.palisade.policy.IsTextResourceRule;
 import uk.gov.gchq.palisade.service.request.Policy;
 
 import java.io.IOException;
-import java.util.stream.Collectors;
-import java.util.stream.StreamSupport;
 
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.core.Is.is;
-import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
 
-@RunWith(JUnit4.class)
+@SpringBootTest
```

Comment:
Does this need to be so heavyweight? Why not use just a standard unittest?

Reply:
removed